src/master-playlist-controller.js

```diff
@@ -1725,11 +1728,23 @@ export class MasterPlaylistController extends videojs.EventTarget {
    * @return {Number} Desired buffer low water line in seconds
    */
   bufferLowWaterLine() {
-    const currentTime = this.tech_.currentTime();
+    const elapsedTime = this.elapsedSinceStart();
     const initial = Config.BUFFER_LOW_WATER_LINE;
     const rate = Config.BUFFER_LOW_WATER_LINE_RATE;
     const max = Math.max(initial, Config.MAX_BUFFER_LOW_WATER_LINE);
 
-    return Math.min(initial + currentTime * rate, max);
+    return Math.min(initial + elapsedTime * rate, max);
+  }
+
+  /**
+   * Calculates the time elapsed since last seek or initial start
+   *
+   * @return {Number} Elapsed time
+   */
+  elapsedSinceStart() {
+    const currentTime = this.tech_.currentTime();
+    const elapsedTime = currentTime - this.startTime_;
+
+    return elapsedTime;
```

Comment:
Preference, but the function could just be `return this.tech_.currentTime() - this.startTime_`

Reply:
Good point, not sure why I did it like that.